snackbar/snackBar.ets

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
Instead of creating 4 different components, create a single component and handle different snackbar features using model class